app/build.gradle

```diff
@@ -117,10 +121,18 @@ dependencies {
     compile 'com.squareup.retrofit:converter-gson:2.0.0-beta2'
     compile 'com.squareup.retrofit:retrofit:2.0.0-beta2'
     compile 'com.squareup.okhttp:logging-interceptor:2.6.0'
+    testCompile 'com.squareup.okhttp:mockwebserver:2.5.0'
+    compile 'com.nispok:snackbar:2.10.8'
+    compile 'com.android.support:appcompat-v7:23.1.0'
+    compile 'com.squareup.okhttp:okhttp:2.5.0'
+    compile 'com.google.code.gson:gson:2.3'
+    compile 'com.squareup.retrofit:converter-gson:2.0.0-beta2'
+    compile 'com.squareup.retrofit:retrofit:2.0.0-beta2'
```

Comment:
yes, 
com.squareup.okhttp:mockwebserver,
com.nispok:snackbar,
com.android.support:appcompat,
com.squareup.okhttp:okhttp:2.5.0, 
com.google.code.gson:gson:2.3,
com.squareup.retrofit:converter-gson
com.squareup.retrofit:retrofit

Are all duplicate.

You only need to leave ar.com.hjg:pngj :-)